Cubical/Foundations/Function.agda

```diff
@@ -26,12 +26,16 @@ _$_ : ∀ {ℓ ℓ'} {A : Type ℓ} {B : A → Type ℓ'} → ((a : A) → B a)
 f $ a = f a
 {-# INLINE _$_ #-}
 
-infixr 9 _∘_
+infixr 9 _∘_ _∘S_
 
 _∘_ : (g : {a : A} → (b : B a) → C a b) → (f : (a : A) → B a) → (a : A) → C a (f a)
 g ∘ f = λ x → g (f x)
 {-# INLINE _∘_ #-}
 
+_∘S_ : (A' → A'') → (A → A') → A → A''
+g ∘S f = λ x → g (f x)
+{-# INLINE _∘S_ #-}
+
```

Comment:
Why do you need that?

Reply:
The name follows the convention from this comment in prelude:
```
{- `S` stands for simply typed. Using `congS` instead of `cong`
   can help Agda to solve metavariables that may otherwise remain unsolved.
-}
```
Similarly, this, non dependent composition helps avoid unsollved metas when taking section,
example from this PR is here:
https://github.com/agda/cubical/blob/491c712ed209308bcc2ab1bc1c7dec871435f764/Cubical/Data/List/Properties.agda#L341
if you replace it with dependent version, unsollved meta apears.

I must admit that I am usiing it by default even it is not necessary (I tend to use less plymorfic functions where possible), I think that this might be confusing, and I can leave it only where it is neessary.

Reply:
I think it's ok.